datacenter/dtr/2.2/guides/admin/configure/garbage-collection.md

```diff
@@ -0,0 +1,97 @@
+---
+description: Configure garbage collection in Docker Trusted Registry
+keyworkds: docker, registry, garbage collection, gc, space, disk space
+title: Docker Trusted Registry 2.2 Garbage Collection
+---
+
+## TL;DR
+
+1. Garbage Collection (GC) reclaims disk space from your storage by deleting
+unused layers
+2. GC can eb configured to run automatically with a cron schedule, and can also
+be run manually; only admins can configure these
+3. When GC runs DTR will be placed in read-only mode; pulls will work but pushes
+will fail.
+4. The UI will show when GC is running, and an admin can stop GC within the UI
+
+**Important notes**
+
+The GC cron schedule is set to run in **UTC time**.  Containers typically run in
+UTC time (unless the system time is mounted), therefore remember that the cron
+schedule will run based off of UTC time when configuring.
+
+GC puts DTR into read only mode; pulls succeed while pushes fail.  This is due
```

Comment:
"This is due to the stop the world" this is a really powerful feature then 😂.
In your head "mark and sweep" is very clear but for most users it won't be. That's why I would move these "important notes" to the end.

You're explaining things that might be very important but you still didn't taught me a foundation. You do that on the "background: how images are stored".